static/src/javascripts/projects/common/modules/identity/manage-account.js

```diff
@@ -5,8 +5,10 @@
 
 import bean from 'bean';
 import reqwest from 'reqwest';
```

Comment:
Would we be able to replace the `reqwest` dependency with `lib/fetch`, which follows the WHATWG `fetch` API? Don't worry if it's too much work

Reply:
i had no idea we had that!! no yeah defo replacing it, at least on the checkboxes (the newsletter options are getting an overhaul and the whole file is gonna be trimmed and that part gone so it's not worth it imo)

Reply:
nevermind https://github.com/guardian/frontend/issues/18244

Reply:
Ah. Okay, thanks for raising the issue!

Reply:
polyfill.io will fill fetch for us too https://polyfill.io/v2/docs/features/#fetch

Reply:
neat, so just using fetch() without importing the existing polyfill will work right?

Reply:
[I don't think we're including fetch](https://assets.guim.co.uk/polyfill.io/v2/polyfill.min.js?rum=0&features=es6,es7,es2017,default-3.6,HTMLPictureElement&flags=gated&callback=guardianPolyfilled&ua=qwerty&unknown=polyfill) at the moment? Perhaps we ought to, then we can kill our polyfill (and hopefully `reqwest` with it)

Reply:
i'm down! let's continue this discussion over at the issue i raised tho bc this got way out of scope for this pr alone 😂